### PR TITLE
ISPN-7043 Add option to normalize cache names

### PR DESCRIPTION
**Pull request by vjuranek**
https://issues.jboss.org/browse/ISPN-7043

### CODE REVIEW

File: src/main/java/org/infinispan/persistence/cloud/CloudStore.java [hunk 1]
```diff
@@ -116,7 +119,7 @@ public void start() {
                .description(String.format("Infinispan cache store for %s", containerName))
                .build();
          blobStore.createContainerInLocation(location, containerName);
-
+
```
Comment (tristantarrant) [inline src/main/java/org/infinispan/persistence/cloud/CloudStore.java]: Some unnecessary whitespace

Comment (vjuranek) [inline src/main/java/org/infinispan/persistence/cloud/CloudStore.java]: done, PR updated

**Comment by tristantarrant** [comment]
Rebase needed

**Comment by vjuranek** [comment]
rebased
